package.json

```diff
@@ -1,30 +1,31 @@
 {
   "name": "@symphony-ui/uitoolkit",
-  "private": true,
   "version": "4.0.7",
+  "private": true,
   "description": "React components for the Symphony UI design system",
+  "license": "ISC",
+  "author": "Symphony",
   "workspaces": [
     "packages/*"
   ],
-  "publishConfig": {
-    "registry": "https://registry.npmjs.org"
-  },
-  "author": "Symphony",
-  "license": "ISC",
   "scripts": {
     "dedupe": "npx yarn-deduplicate yarn.lock",
-    "prepare": "husky install",
-    "postinstall": "yarn workspace @symphony-ui/uitoolkit-styles build"
+    "postinstall": "yarn workspace @symphony-ui/uitoolkit-styles build",
+    "prepare": "husky install"
   },
-  "engines": {
-    "node": ">=16.14.0"
+  "lint-staged": {
+    "*.{ts,tsx,js,jsx}": "eslint --cache --fix",
+    "*.{js,css,md}": "prettier --write"
   },
+  "dependencies": {},
   "devDependencies": {
     "husky": "^8.0.3"
   },
-  "lint-staged": {
-    "*.{ts,tsx,js,jsx}": "eslint --cache --fix",
-    "*.{js,css,md}": "prettier --write"
```

Comment:
Are you sure this shoud be removed ?

Reply:
It didn't get removed, I just ran `https://www.npmjs.com/package/sort-package-json` when adding the `packageManager` field.